### PR TITLE
SDKs: fix SSR

### PR DESCRIPTION
**Pull request by samijaber**
## Description

Pending release and deploy of https://github.com/BuilderIO/mitosis/pull/150

Changes:

- upgrade to latest version of mitosis-cli
- includes an old fix in core mitosis that removes bad/unused `useContext` and `useState` imports for react/react-native
- allow for run-time mitosis configs to include target-specific options
- fix vue SDK not registering components at run-time due to absolute path import
- chore: remove a handful of redundant repo configuration

### CODE REVIEW

File: packages/sdks/output/react-native/src/blocks/button.lite.tsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
 import { View, StyleSheet, Image, Text } from 'react-native';
-import { useContext } from 'react';
```
Comment (samijaber) [inline packages/sdks/output/react-native/src/blocks/button.lite.tsx:3]: These went away when updating to the new mitosis. They were bad/unused imports to begin with

File: packages/sdks/output/react-native/src/components/render-block.lite.tsx [hunk 2]
```diff
@@ -114,7 +118,7 @@ export default function RenderBlock(props) {
       ) : (
         <ComponentRefRef
           {...componentInfo?.()?.options}
-          attributes={properties()}
+          attributes={propertiesAndActions()}
```
Comment (samijaber) [inline packages/sdks/output/react-native/src/components/render-block.lite.tsx:121]: This is from #670

File: packages/sdks/output/vue/nuxt2/src/blocks/button.vue [hunk 3]
```diff
@@ -11,7 +11,7 @@
   <span v-bind="attributes" v-else="">{{ text }}</span>
 </template>
 <script>
-import { registerComponent } from '@builder.io/sdk-vue';
+import { registerComponent } from '../functions/register-component';
```
Comment (samijaber) [inline packages/sdks/output/vue/nuxt2/src/blocks/button.vue:14]: This is the fix!

File: packages/sdks/package.json [hunk 4]
```diff
@@ -1,28 +1,21 @@
...
     "ci:test": "npm run test"
   },
-  "exports": {
-    "./react/*": "./dist/react/src/*.js",
-    "./vue/*": "./dist/vue/src/*.js",
-    "./svelte/*": "./dist/svelte/src/*.js",
-    "./angular/*": "./dist/angular/src/*.js",
-    "./html/*": "./dist/html/src/*.js",
-    "./solid/*": "./dist/solid/src/*.js"
-  },
```
Comment (samijaber) [inline packages/sdks/package.json:19]: These were unused/didn't make sense

**Review by adamdbradley — APPROVED** [review]

**Pull request merged**